remoteworkitem/flatten_test.go

```diff
@@ -0,0 +1,25 @@
+package remoteworkitem
+
+import (
+	"encoding/json"
+	"testing"
+
+	"github.com/almighty/almighty-core/resource"
+)
+
+func TestFlattenMap(t *testing.T) {
+	resource.Require(t, resource.UnitTest)
+	testString := []byte(`{"name":"shoubhik","assignee":{"1":"sbose","2":"pranav","participants":{"4":"sbose56","5":"sbose78"}},"name":"shoubhik"}`)
+	var nestedMap map[string]interface{}
+	err := json.Unmarshal(testString, &nestedMap)
+
+	if err != nil {
+		t.Error("Incorrect dataset ", testString)
+	}
+
+	OneLevelMap := Flatten(nestedMap)
+
+	if OneLevelMap["assignee.participants.4"] != "sbose56" {
```

Comment:
Try using `assert.Equal(t, "sbose56", OneLevelMap["assignee.participants.4"])` to shorten tests like this. Not only is this shorter and easier to write but you'll get much better error messages this way. There are [many more functions](https://godoc.org/github.com/stretchr/testify/assert) in the `assert` package like `assert.NotNil` and so forth.

You'll have to import "`"github.com/stretchr/testify/assert"`" for this to work. It is a dependency that's already vendored.